strapi/strapi
# clean config files for generated projects

### What does it do?

It allows the creation of correct config files for new projects

### Why is it needed?

An unintended git merge partially reverted this PR https://github.com/strapi/strapi/pull/12287

### How to test it?

- Create a new project with
  - `./packages/cli/create-strapi-app/index.js appjs` for JS
  - `./packages/cli/create-strapi-app/index.js appts --typescript` for TS
- See that the config files are correct

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/pull/13910
https://github.com/strapi/strapi/pull/13934
